### PR TITLE
Remove image segmentation duplicates

### PR DESCRIPTION
**Pull request by akansha1812**
Reuse demo/image-segmentation functions in /demo/lightning/image-segmentation.
- unet3d.py
- losses.py
- layers.py is part of unet3d.py
- pytorch/get_train_transforms 
- pytorch/RandBalancedCrop

Tested complete run for /demo/lightning/image-segmentation/train.py. Tested imports for demo/image-segmentation/train.py.

### CODE REVIEW

**Review by awonak — APPROVED** [review]

File: setup.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
     packages=[
         "dataflux_core", "dataflux_pytorch", "dataflux_pytorch.lightning",
         "dataflux_pytorch.multipart_upload", "demo.lightning.text_based",
-        "demo.lightning.checkpoint.multinode"
+        "demo.lightning.checkpoint.multinode","demo.image_segmentation"
```
Comment (MattIrv) [inline setup.py:16]: Maybe not for this PR but I'm not sure the demo code actually needs to be included here. Do things break if we remove it?

**Review by MattIrv — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (MattIrv) [reply setup.py:16]: Can we just use `-m`? In general we should be using `-m`

Comment (akansha1812) [reply setup.py:16]: I think we can do that. Along with Kokoro we have to update some of the readme too. 
I'll create a new PR to remove demo from setup.py

**Review by abhibyreddi — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (akansha1812) [reply setup.py:16]: Resolving the comment for now.

**Pull request merged**